scripts/zkai_attention_kv_softmax_table_proof_byte_accounting_gate.py

```
@@ -0,0 +1,1113 @@
+        "controlled_grid_status",
+        "profile_rows",
+        "scaling_diagnostics",
+        "non_claims",
+        "validation_commands",
+        "accounting_gate_commitment",
+    }
+    mutation_summary_keys = {"mutation_cases", "mutations_checked", "mutations_rejected", "all_mutations_rejected"}
+    if not allow_missing_mutation_summary:
+        expected_keys |= mutation_summary_keys
+    elif mutation_summary_keys & set(payload):
+        expected_keys |= mutation_summary_keys
```

Ruff (0.15.12)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)